public/app/plugins/datasource/elasticsearch/specs/query_builder.test.ts

```diff
@@ -763,16 +763,12 @@ describe('ElasticQueryBuilder', () => {
                 extended_bounds: { max: '$timeTo', min: '$timeFrom' },
```

Comment:
since you are touching this file and we removed support for anything <7.10,  i think it would be nice to change (at the top of this file this:

```ts
  const builder = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '2.0.0' });
  const builder5x = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '5.0.0' });
  const builder56 = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '5.6.0' });
  const builder6x = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '6.0.0' });
  const builder7x = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '7.0.0' });
  const builder77 = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '7.7.0' });
  const builder8 = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '8.0.0' });

  const allBuilders = [builder, builder5x, builder56, builder6x, builder7x, builder77, builder8];
```

to 
```
  const builder710 = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '7.10.0' });
  const builder8 = new ElasticQueryBuilder({ timeField: '@timestamp', esVersion: '8.0.0' });

  const allBuilders = [builder710, builder8];
```

wdyt?

Reply:
in general i agree, but i did the change and now got a bunch of errors below :)  (some tests are done on `builder`, which is now gone, same for `builder7x`... i don't know right now if i should just delete the `builder7x` tests or adjust them etc.)

in short ,it seems some more work is needed in `query_builder.test.ts`, so if you're ok with it, i'd let it be for a later time.